packages/@react-spectrum/list/docs/ListView.mdx

```diff
@@ -0,0 +1,476 @@
+Dynamic collections, as shown below, can be used when the options come from an external data source such as an API, or update over time. Providing the data dynamically allows ListView
+to automatically cache the rendering of each item, which dramatically improves performance.
+
+Each item has a unique key defined by the data. In the example below, the `key` of each row element is implicitly defined by the id property of the row object.
+See [collections](../react-stately/collections.html#unique-keys) to learn more keys in dynamic collections.
+
+
+```tsx example
+const items = [
+  {id: 1, name: 'Adobe Photoshop'},
+  {id: 2, name: 'Adobe XD'},
+  {id: 3, name: 'Adobe InDesign'},
+  {id: 4, name: 'Adobe AfterEffects'},
+  {id: 5, name: 'Adobe Illustrator'},
+  {id: 6, name: 'Adobe Lightroom'},
+  {id: 7, name: 'Adobe Premiere Pro'},
+  {id: 8, name: 'Adobe Fresco'},
+  {id: 9, name: 'Adobe Dreamweaver'}
+];
+
+<ListView items={items} selectionMode="multiple" maxWidth="size-6000" height="250px" aria-label="List example">
+  {(item) => (
+    <Item key={item.key}>
+      {item.name}
+    </Item>
+  )}
+</ListView>
+```
+
+### Internationalization
+To internationalize a ListView, all text content within the ListView should be localized. This includes the `aria-label` provided to the ListView if any.
+For languages that are read right-to-left (e.g. Hebrew and Arabic), the layout of ListView is automatically flipped.
+
+## Labeling
+### Accessibility
+An `aria-label` must be provided to the ListView for accessibility. If the ListView is labeled by a separate element, an `aria-labelledby` prop must be provided using the id of the labeling element instead.
+
+## Asynchronous loading
+ListView supports loading data asynchronously, and will display a progress circle reflecting the current load state,
+set by the `loadingState` prop. It also supports infinite scrolling to load more data on demand as the user scrolls, via the `onLoadMore` prop.
+
+This example uses the [useAsyncList](../react-stately/useAsyncList.html) hook to handle loading the data.
+See the docs for more information.
+
+```tsx example
+import {useAsyncList} from '@react-stately/data';
+
+function AsyncList() {
+
+  let list = useAsyncList({
+    async load({signal, cursor}) {
+      if (cursor) {
+        cursor = cursor.replace(/^http:\/\//i, 'https://');
+      }
+
+      let res = await fetch(cursor || `https://swapi.py4e.com/api/people/?search=`, {signal});
+      let json = await res.json();
+
+      return {
+        items: json.results,
+        cursor: json.next
+      };
+    }
+  });
+
+  return (
+    <ListView
+      selectionMode="multiple"
+      aria-label="example async loading list"
+      maxWidth="size-6000"
+      height="size-3000"
+      items={list.items}
+      loadingState={list.loadingState}
+      onLoadMore={list.loadMore}>
+      {(item) => (
+        <Item key={item.name}>{item.name}</Item>
+      )}
+    </ListView>
+  );
+}
+```
+
+## Complex items
```

Comment:
![Screen Shot 2022-05-23 at 11 20 08 AM](https://user-images.githubusercontent.com/698229/169882546-e7a5ca86-cc0f-4f98-8799-90427ba3d0a5.png)
should has children be disabled?

Reply:
Yeah, it is disabled since this ListView doesn't have an onAction provided. Probably kinda weird, I'll add a dummy onAction to it

Reply:
Sounds good, I'll do this in a followup PR